spec/index.bs

```diff
@@ -3910,6 +3997,56 @@ dictionary GPURenderPassColorAttachmentDescriptor {
 };
 </script>
 
+<dl dfn-type=dict-member dfn-for=GPURenderPassColorAttachmentDescriptor>
+    : <dfn>attachment</dfn>
+    ::
+        A {{GPUTextureView}} describing the texture [=subresource=] that will be output to for this
+        color attachment.
+
+    : <dfn>resolveTarget</dfn>
+    ::
+        A {{GPUTextureView}} describing the texture [=subresource=] that will receive the resolved
+        output for this color attachment if {{GPURenderPassColorAttachmentDescriptor/attachment}} is
+        multisampled.
+
+    : <dfn>loadValue</dfn>
+    ::
+        If a {{GPULoadOp}}, indicates the load operation to perform on
+        {{GPURenderPassColorAttachmentDescriptor/attachment}} prior to executing the render pass.
+        If a {{GPUColor}}, indicates the value to clear {{GPURenderPassColorAttachmentDescriptor/attachment}}
+        to prior to executing the render pass.
+
+    : <dfn>storeOp</dfn>
+    ::
+        The store operation to perform on {{GPURenderPassColorAttachmentDescriptor/attachment}}
+        after executing the render pass.
+</dl>
+
+<div class=validusage dfn-for=GPURenderPassColorAttachmentDescriptor>
+    <dfn abstract-op>Valid Usage</dfn>
+
+    Given a {{GPURenderPassColorAttachmentDescriptor}} |this| the following validation rules
+    apply:
+
+    1. |this|.{{GPURenderPassColorAttachmentDescriptor/attachment}} must have a renderable color format.
+    1. |this|.{{GPURenderPassColorAttachmentDescriptor/attachment}}.{{GPUTexture/[[textureUsage]]}}
+        must contain {{GPUTextureUsage/OUTPUT_ATTACHMENT}}.
+    1. |this|.{{GPURenderPassColorAttachmentDescriptor/attachment}} must be a view of a single [=subresource=].
+    1. If |this|.{{GPURenderPassColorAttachmentDescriptor/resolveTarget}} is not `null`:
+
+        1. |this|.{{GPURenderPassColorAttachmentDescriptor/attachment}} must be multisampled.
+        1. |this|.{{GPURenderPassColorAttachmentDescriptor/resolveTarget}} must not be multisampled.
+        1. |this|.{{GPURenderPassColorAttachmentDescriptor/resolveTarget}}.{{GPUTexture/[[textureSize]]}}
+            must match |this|.{{GPURenderPassColorAttachmentDescriptor/attachment}}.{{GPUTexture/[[textureSize]]}}.
+        1. |this|.{{GPURenderPassColorAttachmentDescriptor/resolveTarget}}.{{GPUTexture/[[format]]}}
+            must match |this|.{{GPURenderPassColorAttachmentDescriptor/attachment}}.{{GPUTexture/[[format]]}}.
+        1. Issue: Describe any remaining resolveTarget validation
+
+    1. Otherwise |this|.{{GPURenderPassColorAttachmentDescriptor/attachment}} must not be multisampled.
```

Comment:
It should be allowed, you can render to a multisampled texture and store it at the end of the render pass.